scipy/_lib/tests/test__util.py

```diff
@@ -22,6 +23,11 @@
 from scipy import cluster, interpolate, linalg, optimize, sparse, spatial, stats
 
 
+lazy_xp_function(_contains_nan, static_argnames=("nan_policy", "xp_omit_okay", "xp"))
+# FIXME @jax.jit fails: complex bool mask
+lazy_xp_function(_lazywhere, jax_jit=False, static_argnames=("f", "f2"))
```

Comment:
It should be easy enough to search for all instances of `jax_jit=False` in the codebase to figure out what is still not working

Reply:
The purpose of this syntax, as opposed to either
- commenting out the whole line, or
- not writing the line at all

is for forward compatibility with Dask, so that we can choose to disable jitting on jax while keeping the compute guard on dask, or vice versa.

Reply:
Seems reasonable. If the line wasn't written at all, then the next person looking at this function would be rediscovering the problem probably.

Is it a FIXME for this PR, or a future one?

Reply:
Future. I intend to move `_lazywhere` to array-api-extra and fix it for good. A fairly sizeable change by itself.